tests/conftest.py

```diff
@@ -80,6 +80,14 @@ def tmpfile(tmpdir):
     return tmpdir / tmpfile
 
 
+@pytest.fixture()
+def tmpmodule(tmpdir):
+    tmpfile = f'package.py'
+    with open(tmpdir / tmpfile, 'w') as f:
+        f.write("raise ImportError")
+    return tmpdir / tmpfile
+
+
```

Comment:
if this fixture is only needed once lets move it inside the test or at least inside the test folder